go.mod

```diff
@@ -133,3 +133,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types v0.0.0-20230221114307-1825fd3fbad7 => ./types
```

Comment:
JFYI, I'm also touching types in https://github.com/aquasecurity/tracee/pull/2845/commits/d6a2a78d7cb6ca3a68acc87dbb75453865ca2fc4.